### DISCUSSION TITLE
What is the national animal emblem of China?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#633] [Galaxy Trivia] [answered]
**Bonus:** How about India and Panama?

*answer with emojis*

**Comment by MohammedNayeem9** [comment, marked as answer]
The national animal emblem of **China 🇨🇳** is the **Giant Panda 🐼**.

The national animal emblem of **India 🇮🇳** is the **Bengal Tiger🐅🐯**.

The national animal emblem of **Panama 🇵🇦** is the **Harpy Eagle 🦅**.

**Comment by peterbaccaro** [comment]
The national animal emblem of China is the **Giant Panda** 🐼.
The national animal emblem of India is the **Bengal Tiger** 🐯.
The national animal emblem of Panama is the **Harpy Eagle** 🦅.

**Reply by MohammedNayeem9** [reply]
India🇮🇳 national emblem animal is four lions🦁. 
National animal of india🇮🇳 is bengal tiger🐅

**Reply by peterbaccaro** [reply]
You're right.

**Comment by DelrosarioRovic** [comment]
china:giant panda 🐼
india:Bengal tiger 🐅
panama: harpy eagle 🦅
